### ISSUE TITLE
Cannot find module 'eslint-plugin-import'

### ISSUE THREAD

**Issue by zeke** [standard/standard#1081]
- PR: https://github.com/electron/electronjs.org/pull/1119
- Failing Test https://travis-ci.org/electron/electronjs.org/builds/343168963

```
standard: Unexpected linter output:
Error: Failed to load plugin import: Cannot find module 'eslint-plugin-import'
Referenced from: /home/travis/build/electron/electronjs.org/node_modules/standard/eslintrc.json
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.resolve (internal/module.js:18:19)
    at Plugins.load (/home/travis/build/electron/electronjs.org/node_modules/eslint/lib/config/plugins.js:106:29)
    at Array.forEach (<anonymous>)
    at Plugins.loadAll (/home/travis/build/electron/electronjs.org/node_modules/eslint/lib/config/plugins.js:146:21)
    at loadFromDisk (/home/travis/build/electron/electronjs.org/node_modules/eslint/lib/config/config-file.js:492:35)
    at load (/home/travis/build/electron/electronjs.org/node_modules/eslint/lib/config/config-file.js:550:20)
    at configExtends.reduceRight (/home/travis/build/electron/electronjs.org/node_modules/eslint/lib/config/config-file.js:421:36)
    at Array.reduceRight (<anonymous>)
    at applyExtends (/home/travis/build/electron/electronjs.org/node_modules/eslint/lib/config/config-file.js:403:28)
If you think this is a bug in `standard`, open an issue: https://github.com/standard/standard/issues
```

**Comment by feross** [comment]
This is super odd. We depend on it in `package.json` [here](https://github.com/standard/standard/blob/c93ac0d56402496f80570288692004443b997804/package.json#L18).

I just cloned the branch and tested locally with the exact same version of Node and npm and it worked fine. I just merged master into that PR branch and re-triggered a build. My guess is that it's a race condition npm bug.

**Comment by feross** [comment]
Didn't work. I'm stumped. npm is not installing the deps:

<img width="548" alt="screen shot 2018-02-27 at 8 50 27 pm" src="https://user-images.githubusercontent.com/121766/36770585-eae65718-1bff-11e8-8786-eff8ed2c64a9.png">

**Comment by zkat** [comment]
I'll look at it in more detail tomorrow but: this looks like it's the peerdep hoisting issue. You'll likely have to manually install the missing peerdep at the top level.

tl:dr: npm knows not to hoist peerdeps in some situations but then it turns out it hoists the thing that requires that peerdep. This is not really trivial to fix because it needs a big change to the installer.

But a workaround is that you can just install a matching version of `es-plugin-import` at the top level and the thing that needs it will now, correctly, have a copy of it.

This bug, afaik, has been around pretty much since npm@3.

**Comment by zeke** [comment]
Closing, as this appears to have been (perhaps indirectly) resolved in standard@11.0.1

See https://github.com/electron/electronjs.org/pull/1209

**Comment by feross** [comment]
@zeke Nice to hear!
